dozer-api/src/generator/oapi/utils.rs

```diff
@@ -140,7 +140,14 @@ fn convert_cache_type_to_schema_type(field_type: dozer_types::types::FieldType)
                 ..Default::default()
             })
         }
-        FieldType::Binary | FieldType::Bson => Type::Array(ArrayType {
+        FieldType::Json => Type::Object(ObjectType {
```

Comment:
Json is not necessarily an object. Is there some kind of `Any`?

Reply:
yes will refactor the function to return the `SchemaKind` instead of `Type` itself as `Type` doesn't have anything that can be able to be mapped into `Any`